### DISCUSSION TITLE
ModuleNotFoundError

### DISCUSSION THREAD

**Discussion by thomasrgrant** [TomSchimansky/CustomTkinter#2462] [Q&A]
import model.ctk_theme_builder as mod
me_builder_qa_app.py", line 9, in <module>
    import model.ctk_theme_builder as mod
ModuleNotFoundError: No module named 'model'

How do I fix this error?

**Comment by dipeshSam** [comment]
Although, this is not Tkinter specific problem, but a sample reproducible code and description of what you are trying to get might be helpful to solve it 

Regards.

**Comment by thomasrgrant** [comment]
![t2](https://github.com/TomSchimansky/CustomTkinter/assets/18148511/4a5203ec-8055-40f4-a2c7-aff71ebec62f)
![mod](https://github.com/TomSchimansky/CustomTkinter/assets/18148511/073c11b0-0c57-4d56-9276-33694edc7550)
Here you can see the issue.

**Comment by dipeshSam** [comment]
Make sure whether the `model` module properly exists or not.

And, you can try the followings:

1. Confirm directory structure, it should look like the following:

``` script

my_project/
├── __init__.py
├── theme_builder_qa_app.py
└── model/
    ├── __init__.py
    └── ctk_theme_builder.py

```

2. Try relative import or import specific variable. Like the following:

``` Python
from .model import ctk_theme_builder as mod

```
Or,
``` Python
from model import ctk_theme_builder as mod
```

It might be helpful.
Regards.
